web/admin/src/App.js

```diff
@@ -274,8 +278,13 @@ class App extends PureComponent {
     }
   }
 
-  loadBuilder = (event, { history }) => {
-    this.setState({ config: event.target.value, configKey: event.target.name, showBuilder: true })
+  loadBuilder = (survey, { history }) => {
```

Comment:
Change function params from `survey, {history}` to `survey, history` (and update callers)